### DISCUSSION TITLE
Need guide in my web development journey!!

### DISCUSSION THREAD

**Discussion by ayusmanarya2001** [cssbattle/forum#217] [Ideas]
Hello everyone this is rahool . I have heard that github communities are very open for beginners. I have started my coding journey last month. I am very new to this front end part and I am very new to github as well as CSSbattle. I will be very much pleased if anyone can guide me how to start participating in css battle dev as beginner and excel  in this journey.

I am hoping for this communities guide and feedback.
Thank you ;)

**Comment by Beowolve** [comment]
I would suggest, just start with some simple targets and ask questions about specific problems if you are stuck somewhere.

However, keep in mind that cssbattle.dev is about how to "not code" right. Because in real world applications / websites you would not code like that. So if you are a pure beginner and want to create useful web pages later, I would suggest looking for some tutorials and first learn how to do things right and not the other way around, otherwise you could establish some bad habits.

**Reply by ayusmanarya2001** [reply]
thank you very much for your response. Yes, I am learning from tutorials and sometimes it gets very difficult for me while doing a mini project . I am trying my best to learn as much thing as possible.
